From 03ff96a12e3aafbb9c042a60e0627b7f532f44bc Mon Sep 17 00:00:00 2001
From: "Gross, Mark" <mark.gross@intel.com>
Date: Fri, 14 Sep 2018 13:33:54 -0700
Subject: [PATCH 1645/1666] fix some likely copy paste errors with some if
 blocks

gcc 7.3 flagged some iffy if statements with python indentation needing
some brackets.  this patch adds them.

Signed-off-by: mark gross<mark.gross@intel.com>
---
 drivers/staging/igb_avb/e1000_phy.c | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/drivers/staging/igb_avb/e1000_phy.c b/drivers/staging/igb_avb/e1000_phy.c
index 0534e4a..46ab8d5 100644
--- a/drivers/staging/igb_avb/e1000_phy.c
+++ b/drivers/staging/igb_avb/e1000_phy.c
@@ -3291,12 +3291,13 @@ s32 e1000_read_phy_reg_mphy(struct e1000_hw *hw, u32 address, u32 *data)
 	*data = E1000_READ_REG(hw, E1000_MPHY_DATA);
 
 	/* Disable access to mPHY if it was originally disabled */
-	if (locked)
+	if (locked){
 		ready = e1000_is_mphy_ready(hw);
 		if (!ready)
 			return -E1000_ERR_PHY;
 		E1000_WRITE_REG(hw, E1000_MPHY_ADDR_CTRL,
 				E1000_MPHY_DIS_ACCESS);
+	}
 
 	return E1000_SUCCESS;
 }
@@ -3356,12 +3357,13 @@ s32 e1000_write_phy_reg_mphy(struct e1000_hw *hw, u32 address, u32 data,
 	E1000_WRITE_REG(hw, E1000_MPHY_DATA, data);
 
 	/* Disable access to mPHY if it was originally disabled */
-	if (locked)
+	if (locked) {
 		ready = e1000_is_mphy_ready(hw);
 		if (!ready)
 			return -E1000_ERR_PHY;
 		E1000_WRITE_REG(hw, E1000_MPHY_ADDR_CTRL,
 				E1000_MPHY_DIS_ACCESS);
+	}
 
 	return E1000_SUCCESS;
 }
-- 
2.7.4

